change/chat-stateful-client-081fed3a-3527-4bfa-b623-7da85ea465f4.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
nit (**that doesn't need fixed**): I'd mark this as a `none` in future, I don't think its necessary for the changelog (always happy to be argued wrong though :D )

Reply:
Makes perfect sense. I wanted to skip changelog, but didn't realize I could choose none.